src/components/search/_search.scss

```diff
@@ -12,10 +12,13 @@ $includeHtml: false !default;
       position: absolute;
       top: 0;
       bottom: 0;
-      right: 6px;
+      right: gutter(0.5);
       display: flex;
       align-items: center;
-      padding-right: gutter(0.25);
+    }
+
+    &--short &__button {
```

Comment:
input that we put into a small search box is called mint-input--low, why this is `--short`? I think it should also be `--low`, WDYT?